doc/quickstart/postgis_quickstart.rst

```diff
@@ -399,8 +401,8 @@ them blue for you - use the QGIS documentation to work out how to change this. Z
 a famous group of lakes in Canada.
 
 
-Creating A Spatial Table The Easy Way
-================================================================================
+Creating a spatial table the easy way
+=====================================
 
 Most of the OSGeo desktop tools have functions for importing spatial data in files, such as shapefiles,
 into PostGIS databases. Again we'll use QGIS to show this.
```

Comment:
Please change the sentence to (as spatial data can be in different formats):

Most of the OSGeo desktop tools have functions for importing spatial data from other formats (f.e. ESRI Shape) into the database. Again we'll use QGIS to show this.

Reply:
Fixed